packages/tasit-contracts/test/SimpleStorage.js

```diff
@@ -0,0 +1,15 @@
+const SimpleStorage = artifacts.require("./SimpleStorage.sol");
+
```

Comment:
```suggestion
// Note: It will probably be a point of confusion for new developers
// coming to this project that they'll have to get used to the web3.js
// API for writing truffle tests but then use the ethers.js API and our
// own API also in JavaScript for testing our own code. So the
// ethers.js tests in tasit-act will look different than the truffle tests
// in tasit-contracts testing the same contract.
// For that reason, a possible TODO is removing any truffle tests
// other than those we directly add from 3rd-party projects
```